### PR TITLE
Local files

### PR DESCRIPTION
**Pull request by natechols**

### CODE REVIEW

File: REQUIREMENTS.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
-numpy
-h5py
+cython
+numpy >= 1.7.1
+h5py >= 2.0.1
+pysam >= 0.8.4
```
Comment (pb-cdunn) [inline REQUIREMENTS.txt]: If a version changes, how would you invalidate the cache? Just curious.

Comment (natechols) [inline REQUIREMENTS.txt]: I honestly have no idea - this was cribbed from another project.

Comment (pb-cdunn) [inline REQUIREMENTS.txt]: Ah. I see a "Rebuild without cache" button in CircleCI.

I know that CircleCI works extremely well for Python projects, but C++ is the hard stuff. If caching in CircleCI works reliably for C++ projects, we'll move our TravisCI builds to it. Caching is more important than any of the advantages of TravisCI. (Mainly, huge server farm and fast VM spin-up.)

**Comment by dalexander** [comment]
circle works great for C++.  Caching works fine.

**Comment by pb-cdunn** [comment]
@dalexander, could you point me to a C++ project using CircleCI caching? **pbbam** still uses Travis.

There is one nice thing in TravisCI that we would lose: building with Clang and gcc simultaneously.

**Pull request merged**